pkgs/applications/misc/openrgb/default.nix

```diff
@@ -25,6 +25,9 @@ mkDerivation rec {
 
     mkdir -p $out/etc/udev/rules.d
     cp 60-openrgb.rules $out/etc/udev/rules.d
+    
+    install -Dm444 -t "$out/share/applications" qt/OpenRGB.desktop
+    install -Dm444 -t "$out/share/icons/hicolor/128x128/apps" qt/OpenRGB.png
```

Comment:
```suggestion

    install -Dm444 -t "$out/share/applications" qt/OpenRGB.desktop
    install -Dm444 -t "$out/share/icons/hicolor/128x128/apps" qt/OpenRGB.png
```

Reply:
https://github.com/NixOS/nixpkgs/runs/4973144807?check_suite_focus=true

Reply:
Should be fine now: https://github.com/NixOS/nixpkgs/runs/4983610330?check_suite_focus=true

Thanks!